connector-template/mod.rs

```diff
@@ -42,8 +54,8 @@ impl ConnectorCommon for {{project-name | downcase | pascal_case}} {
     }
 
     fn common_get_content_type(&self) -> &'static str {
-        todo!()
         // Ex: "application/x-www-form-urlencoded"
+        ""
```

Comment:
won't this be hard to debug? imagine someone trying out to integrate connector and everything is running but the content type is wrong so the requests are not working.

Reply:
Keeping "application/json" as the default value.